benchmarks/test_cattrs.py

```diff
@@ -3,7 +3,7 @@
 
 import attr
 import cattr
-from dateutil.parser import parse
+from ciso8601 import parse_datetime as parse
```

Comment:
Not sure what the full scope of the parsing and the support matrix of these benchmarks is, but if you are willing to scope down from "parses all date-like strings" to "parses the subset of ISO 8601 supported by `ciso8601`", then you may also be willing to scope it to "parses the results of `datetime.isoformat()`", in which case in Python 3.7+ there's the `datetime.fromisoformat` alternate constructor, which is very fast (likely same speed as `ciso8601`).

It appears [someone has backported](https://pypi.org/project/backports-datetime-fromisoformat/) it to earlier Python 3 versions as well.

Reply:
Thanks, I wasn't aware of this. I updated the PR to use `datetime.fromisoformat`, and also gave you write access to my fork, in case you'd like to push to this PR.